app/assets/stylesheets/objects/_history.scss

```diff
@@ -1,3 +1,11 @@
+.app-timeline-heading {
+  display: none;
```

Comment:
This would be a great improvement if we do it mobile first and make it available for screen readers too by keeping it in the accessibility tree.
```
.app-timeline-heading {
    @include govuk-media-query($from: tablet) {
        @include govuk-visually-hidden;
    }
}
```